protos/rb/api/joint_operation_service.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package rb.api;
+
+import "rb/api/joint_operation.proto";
+
+service JointOperationService {
+  // Joint command
+  rpc JointOperation(JointOperationRequest) returns (JointOperationResponse) {}
```

Comment:
It would be helpful to separate each operation into its own RPC method with a corresponding, distinct request type. This way, each method has a request structure tailored to its specific needs. Here’s a suggested breakdown:

```grpc
rpc ServoOn(ServoOnRequest) returns (ServoOnResponse); // OPERATION_SERVO_ON
rpc BrakeEngage(BrakeEngageRequest) returns (BrakeEngageResponse); // OPERATION_BRAKE_ENGAGE
rpc BrakeRelease(BrakeReleaseRequest) returns (BrakeReleaseResponse); // OPERATION_BRAKE_RELEASE
rpc HomeOffsetReset(HomeOffsetResetRequest) returns (HomeOffsetResetResponse); // OPERATION_HOME_OFFSET_RST
rpc SetPositionPIDGain(SetPositionPIDGainRequest) returns (SetPositionPIDGainResponse); // OPERATION_SET_GAIN
```